core/ui/compose/demo/src/main/java/app/k9mail/core/ui/compose/theme/ThunderbirdTheme.kt

```diff
@@ -0,0 +1,37 @@
+package app.k9mail.core.ui.compose.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val ThunderbirdLightColorPalette = lightColors(
+    primary = material_blue_600,
+    primaryVariant = material_light_blue_500,
+    secondary = material_pink_500,
+    secondaryVariant = material_pink_300
+)
+
+private val ThunderbirdDarkColorPalette = darkColors(
+    primary = material_blue_100,
+    primaryVariant = material_blue_50,
+    secondary = material_pink_300,
+    secondaryVariant = material_pink_500
+)
```

Comment:
These probably shouldn't start with a capital letter.

Reply:
There are more places, they used to be Composable functions but I changed that and forgot to change the name too.